src/jdk.management/unix/classes/com/sun/management/internal/OperatingSystemImpl.java

```diff
@@ -41,7 +41,10 @@
     implements com.sun.management.UnixOperatingSystemMXBean {
 
     private static final int MAX_ATTEMPTS_NUMBER = 10;
+    private static final int PER_CPU_SHARES = 1024;
```

Comment:
I think this is no longer used anywhere so can get removed.

Reply:
Done